langchain/src/agents/tests/agent.int.test.ts

```diff
@@ -33,7 +35,61 @@ test("Run agent from hub", async () => {
     input:
       "Who is Olivia Wilde's boyfriend? What is his current age raised to the 0.23 power?",
   });
-  console.log(res);
+  console.log(
+    {
+      res,
+    },
+    "Run agent from hub response"
+  );
+  expect(res.output).not.toEqual("");
+  expect(res.output).not.toEqual("Agent stopped due to max iterations.");
```

Comment:
Added these to all tests since in earlier commits they were passing with this message as the response